docs/openapi/schemas.yaml

```diff
@@ -474,6 +481,71 @@ ExperimentationOpportunity:
       description: The metrics for the experimentation opportunity
       items:
         type: object
+URLInspection:
+  type: object
+  readOnly: true
+  properties:
+    indexStatusResult:
+      type: object
+      description: Result of the index status analysis.
+      properties:
+        verdict:
+          $ref: '#/Verdict'
+        lastCrawlTime:
```

Comment:
There's a DataType already defined in schemas: https://github.com/adobe/spacecat-api-service/blob/main/docs/openapi/schemas.yaml#L6